Shared/Samples/Identify features in WMS layer/IdentifyFeaturesInWMSLayerView.swift

```diff
@@ -0,0 +1,112 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct IdentifyFeaturesInWMSLayerView: View {
+    /// A map with a dark gray base basemap centered on the USA.
+    @State private var map: Map = {
+        let map = Map(basemapStyle: .arcGISDarkGrayBase)
+        let center = Point(x: -95.7129, y: 37.0902, spatialReference: .wgs84)
+        map.initialViewpoint = Viewpoint(center: center, scale: 7e7)
+        return map
+    }()
+    
+    /// The WMS layer with EPA water info.
+    @State private var waterInfoLayer: WMSLayer?
+    
+    /// The text of a WMS feature HTML attribute that is shown in the web view.
+    @State private var webViewText = String()
+
+    /// The tapped screen point.
+    @State var tapScreenPoint: CGPoint?
+    
+    /// The string text for the identify layer results overlay.
+    @State var overlayText = "Tap on the map to identify features in the WMS layer."
+
+    /// The error shown in the error alert.
+    @State private var error: Error?
+
+    var body: some View {
+        MapViewReader { mapViewProxy in
+            MapView(map: map)
+                .onSingleTapGesture { screenPoint, _ in
+                    tapScreenPoint = screenPoint
+                }
+                .task {
+                    do {
+                        // Create a WMS layer from a URL and load it.
+                        let wmsLayer = WMSLayer(url: .EPAWaterInfo, layerNames: ["4"])
+                        try await wmsLayer.load()
+                        
+                        // Add the layer to the map.
+                        map.addOperationalLayer(wmsLayer)
+                        
+                        waterInfoLayer = wmsLayer
+                    } catch {
+                        self.error = error
+                    }
+                }
+                .task(id: tapScreenPoint) {
+                    // Identify on WMS layer using the screen point.
+                    webViewText = if let screenPoint = tapScreenPoint,
+                                     let waterInfoLayer,
+                                     // Identify feature on water info layer
+                                     let identifyResult = try? await mapViewProxy.identify(
```

Comment:
Since you are already using an error alert in this sample, would it make sense to catch the error from the identify instead of failing silently?